webapp/src/main/webapp/i18n/vivo_all.properties

```diff
@@ -771,3 +771,42 @@ through_today = Publications through today's date
 doi_link=Digital Object Identifier (DOI)
 
 using_cache_time=Using information cached at
+
+#
+#New Entries for i18n purposes
+#
+#
+edit_entry_for_head_role=: Edit this Head role

```

Comment:
Please be consistent with capitalization. I propose changing 'Head' to lowercase. Same with 'Organizer' below.
Also, minor thing but I think 'Edit this Head role' sounds very odd. I propose 'Edit this "head of" role.'